index.js

```diff
@@ -4,15 +4,65 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "silver"
 
 
 /***** Deliverable 3 *****/
 // console.log('This is what the traveler object looks like: ', traveler)
+// Using the `traveler` object, update the DOM to show the traveler's *name*, *nickname*, and *photo* in the appropriate places. For the `<img>` tag, make sure to update the `src` attribute *and* the `alt` attribute (it's important for accessibility).
+const heading2 = document.querySelector('h2')
+const em = document.querySelector('em')
+const profileImg = document.querySelector('#profile img')
+
+// Update each attributes
+heading2.textContent = traveler.name 
+em.textContent = traveler.nickname
+profileImg.src = traveler.photo
+profileImg.alt = traveler.nickname
```

Comment:
Great work! One recommendation. When you are working with multiple variables referencing tags, it may be easier to give them a more informative name.  E.g. instead of 'em', you could do 'nicknameTag'. But its up to you